web-components/src/components/combobox/ComboBox.ts

```diff
@@ -1191,6 +1205,8 @@ export namespace ComboBox {
             bubbles: true
           })
         );
+      this.listItemsCount = this.filteredGroupOptions.length;
+      this.notifySearchResultCount()
```

Comment:
1. Can you end with ";" at the lines - 803 and 1209.
2. Can you add UTs for new code added.

Reply:
Done

Reply:
<img width="799" alt="Screenshot 2024-03-11 at 3 30 40 PM" src="https://github.com/momentum-design/momentum-ui/assets/62088969/42086160-4245-4b89-b70c-e35bbccd0016">